docs/registry/configure.md

```diff
@@ -68,17 +68,17 @@ If you are using a [rented Viam rover](/get-started/try-viam/), adding modules i
 
 To add a modular [component](/components/) from the Viam registry to your machine:
 
-1. Navigate to the **Config** tab of your machine's page in [the Viam app](https://app.viam.com).
-1. Click on the **Components** subtab and click the **Create component** button.
-1. Browse the list of available component types, and select the specific modular component you'd like to add.
+1. Navigate to the **CONFIGURE** tab of your machine's page in [the Viam app](https://app.viam.com).
+2. Click the **+** icon next to your machine part in the left-hand menu and select **Component**.
+3. Browse the list of available component types, and select the specific modular component you'd like to add.
```

Comment:
just making sure: you know you don't need to change these right? It will automatically render to be the correct numbers

Reply:
Yeah I know-- my vscode does that